CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 5.0.0
+
+### Breaking Changes
+
+- [#71](https://github.com/okta/okta-react/pull/71) Added required prop `restoreOriginalUri` to `Security`
```

Comment:
added -> adds

let's also mention what has been changed to the default behavior here.